paypal_express.yml

```diff
@@ -4,6 +4,10 @@
   :password: <%= ENV['PASSWORD'] %>
   :test: true
 
+:paypal_buyer:
```

Comment:
Since this is only for testing, I don't think we need it in the YAML file (it can be confusing too for the user).

Instead, how about:

* Referring to `ENV['BUYER_USERNAME']` and `ENV['BUYER_PASSWORD']` directly in `browser_helpers.rb`
* If any of them is `blank`, default to the previous method `$stdin.gets` (with a helpful message)